README.md

```diff
@@ -26,4 +26,14 @@ This will require installing `pre-commit` if you have not already done so.
 
 ## Cloud Development
 
-Currently we do not have a staging or production stack, but a development stack can be deployed with the [infrastrucutre README](infrastructure/README.md)
+Currently we do not have a staging or production stack, but a development stack can be deployed with the [infrastrucutre READEME](infrastructure/README.md)
+
+## Testing
+
+To populate the dev database with some realistic data, please run:
+
+```python
+cd api && docker-compose run --rm web python3 manage.py populate_test_database
```

Comment:
Ariel moved docker-compose up to top level in his last PR. 
```suggestion
docker-compose run --rm web python3 manage.py populate_test_database
```

Reply:
There might be some changes I have to make to relative paths in the script for that command to work. I'll hold off until Ariel's changes are working and then make this modification.

Reply:
Changes have been made.

Reply:
Done.